data-intake/ddi/extension-table/jupyter/name-rank-sample.ipynb

```diff
@@ -16,17 +16,15 @@
     "\n",
     "### Prerequisites\n",
     "Before running this sample, do the following:\n",
-    "1. In your Visier tenant, create a simple numeric property with the object name `Name_Rank` on the `Employee` object in Visier. Publish the property to production.\n",
+    "1. In your Visier tenant, create a simple number property with the object name `Name_Rank` on the `Employee` object in Visier. Publish the property to production.\n",
     "1. In your Visier tenant, create a profile and assign it to your API user with the following capabilities and additional capabilities:\n",
     "   1. Data: Write, API\n",
     "   1. Model: Read, API\n",
     "   1. Direct Publish\n",
     "   1. Legacy API Access\n",
     "   1. Manage Jobs\n",
     "   1. Upload Data\n",
-    "1. Defined an `.env` file as described in the [Python Connector documentation](https://github.com/visier/connector-python#jupyter-basic-authentication-example)\n",
-    "1. Downloaded the [National Data file](https://www.ssa.gov/oact/babynames/names.zip) and extract file `yob2022.txt`.\n",
-    "1. Copy `yob2022.txt` into a directory named `data`."
+    "1. Defined an `.env` file as described in the [Python Connector documentation](https://github.com/visier/connector-python#jupyter-basic-authentication-example)"
```

Comment:
This refers to a component that we will be deprecating. Better to move the content in here or refer to a sample/SDK readme that contains this information

Reply:
Changed comment for
Defined an `.env` file as described in the `.env.extension-table`